style.css

```diff
@@ -148,6 +148,428 @@ body {
+}
+
+.form__colors-item:nth-of-type(4)::before {
+  background: #90bd8f;
+}
+
+.middle-section__colors-and-price {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin: 0;
+  height: 46px;
+  min-width: 250px;
+  margin-bottom: 22px;
+}
+
+.middle-section__order {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding: 0;
+}
+
+.middle-section__order-item {
+  width: 256px;
+  padding-top: 9px;
+  padding-bottom: 26px;
+  margin-bottom: 10px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  text-align: center;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  cursor: pointer;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+}
+
+.middle-section__price {
+  margin: 0;
+  margin-right: 8px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.middle-section__subtitle {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 10px;
+  letter-spacing: .07em;
+  color: #000;
+  opacity: .5;
+}
+
+button {
+  background: none;
+}
+
+.middle-section__btn,
+.middle-section__btn_other {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.middle-section__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 20px;
+  color: #000000;
+}
+
+.middle-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.bottom-section {
+  position: relative;
+  padding: 33px 12px;
+  background: -webkit-gradient(linear, left top, right top, color-stop(27%, #ebe3d8), color-stop(27%, #000), color-stop(38%, #000), color-stop(38%, #ebe3d8));
+  background: -webkit-linear-gradient(left, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+  background: -o-linear-gradient(left, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+  background: linear-gradient(to right, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+}
+
+.bottom-section__image {
+  z-index: -50;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.bottom-section__img {
+  width: 100%;
+  -webkit-box-shadow: -11px 25px 40px -34px rgba(0, 0, 0, .8);
+  box-shadow: -11px 25px 40px -34px rgba(0, 0, 0, .8);
+}
+
+.bottom-section__img img {
+  width: 100%;
+  height: 100%;
+  -o-object-fit: contain;
+  object-fit: contain;
+}
+
+.bottom-section__block {
+  z-index: 50;
+  position: relative;
+  top: -24px;
+  margin-left: 32px;
+  margin-right: 14px;
+  padding: 20px 25px;
+  background: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)